server/enterprise/message_export/shared/export_data.go

```diff
@@ -64,7 +65,7 @@ type ChannelExport struct {
 	ChannelName  string
 	DisplayName  string
 	StartTime    int64 // utc timestamp (milliseconds), start of export period or create time of channel, whichever is greater.
-	EndTime      int64 // utc timestamp (seconds), end of export period or delete time of channel, whichever is lesser. Example: 1366611728.
+	EndTime      int64 // utc timestamp (milliseconds), end of export period or delete time of channel, whichever is lesser. Example: 1366611728.
```

Comment:
Should we update the example value to be in milliseconds?

Reply:
I'll just remove it, it's a useless example. :)